src/multiStepInput.ts

```diff
@@ -0,0 +1,185 @@
+// https://github.com/microsoft/vscode-extension-samples/blob/master/quickinput-sample/src/multiStepInput.ts
```

Comment:
This class is entirely copied from a sample provided by Microsoft, which is unfortunately unavailable as an npm package.